### PR TITLE
hotfix/Update livenessProbe and readinessProbe path for 2.4.1 version of docgen.

### PR DESCRIPTION
**Pull request by areyeslo**
# Main
- Update path in probes for docgen after upgrade.

# Other
- N/A

# How to test
- N/A

# Notes
-N/A

### CODE REVIEW

**Review by jon-funk — DISMISSED** [review]
Thank you for the fix Arturo!

File: openshift4/templates/docgen.dc.yaml [hunk 1]
```diff
@@ -120,7 +120,7 @@ objects:
               livenessProbe:
                 failureThreshold: 3
                 httpGet:
-                  path: /health
+                  path: api/v2/health
```
Comment (BenHarrington) [inline openshift4/templates/docgen.dc.yaml]: I think there needs to be a leading slash for both the paths

Comment (areyeslo) [reply openshift4/templates/docgen.dc.yaml]: thanks Ben!!

**Review by BenHarrington — APPROVED** [review]
Thanks for updating that Arturo!

**Pull request merged**